charts/datasaur/values.yaml

```diff
@@ -162,145 +162,43 @@ jobs:
-                            type: last
-                          type: query
-                    datasource:
-                        type: __expr__
-                        uid: __expr__
-                    expression: A
-                    hide: false
-                    intervalMs: 1000
-                    maxDataPoints: 43200
-                    reducer: mean
-                    refId: B
-                    settings:
-                        mode: dropNN
-                    type: reduce
-                - refId: C
-                  relativeTimeRange:
-                    from: 600
-                    to: 0
-                  datasourceUid: __expr__
-                  model:
-                    conditions:
-                        - evaluator:
-                            params:
-                                - 1
-                            type: gt
-                          operator:
-                            type: and
-                          query:
-                            params:
-                                - C
-                          reducer:
-                            params: []
-                            type: last
-                          type: query
-                    datasource:
-                        type: __expr__
-                        uid: __expr__
-                    expression: B
-                    hide: false
-                    intervalMs: 1000
-                    maxDataPoints: 43200
-                    refId: C
-                    type: threshold
-              noDataState: OK
-              execErrState: Error
-              for: 1m
-              annotations:
-                description: |-
-                    Node: {{ $labels.node_name }}
-                    Pod: {{ $labels.pod }}
-                summary: |-
-                    {{ $labels.alertname }}
-                    {{ $labels.app }} has more than 5 error request in the last 10 minute
-              labels:
-                client: datasaur
-              isPaused: false
-    notification.yaml: |
-      apiVersion: 1
-
-      policies:
-        - orgId: 1
-          receiver: cp_1
-          matchers: 
-            - client = datasaur
-          object_matchers:
-            - ['client', '=', 'datasaur']
+  alerting: null
 
   auth:
     username: null
     password: null
+  
+  loki:
+    config:
+      common:
+        path_prefix: /var/loki
+        replication_factor: 1
+
+      schema_config:
+        configs:
+          - from: "2022-01-11"
+            index:
+              period: 24h
+              prefix: loki_index_
+            store: boltdb-shipper
+            object_store: aws
+            schema: v12
+
+      storage_config:
+        boltdb_shipper:
+          shared_store: s3
+        aws:
+          s3: s3://us-east-1/datasaur-test-bucket
+          access_key_id: AKIA6GZQPKT63SAO5MWM
+          secret_access_key: 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
```

Comment:
are these real aws keys?

credentials ga boleh ditaro di github repository, apalagi public

Reply:
ah sorry, forgot to remove